misk/src/main/kotlin/misk/web/actions/StatusAction.kt

```diff
@@ -0,0 +1,27 @@
+package misk.web.actions
+
+import misk.healthchecks.HealthCheck
+import misk.web.Get
+import misk.web.Response
+import misk.web.ResponseBody
+import okio.BufferedSink
+import java.io.File
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class StatusAction : WebAction {
+    @Inject lateinit var healthChecks: MutableSet<HealthCheck>
+
+    @Get("/_status")
```

Comment:
We want to reserve ``_status`` for returning machine parsable status messages.  In general, unadorned paths starting underscore should be reserved for control plane interactions.

We should place admin related UI pages under an admin specific top-level namespace (e.g. `_ah/`)